MainBrain/src/Model/GasPedal/GasPedal.hpp

```diff
@@ -22,6 +22,8 @@ class GasPedal : public BasePedal
 
     void setRawOrigin(void);
 
+    const int MAX_ANALOGREAD = 8191;    //(2^13)-1
```

Comment:
This will be common among all classes that use analogRead(). I would suggest creating a file called `Constants.hpp` under the folder `src/Model/Constants/` and putting this in that new file. 

That newly created file would have to be included into the `BaseModel.hpp` file under `src/Model/BaseModel/` with something like `#include "../Constants/Constants.hpp"`